lib/utils.pm

```diff
@@ -2300,7 +2300,7 @@ sub permit_root_ssh_in_sol {
     my $sshd_config_file = shift;
 
     $sshd_config_file //= "/etc/ssh/sshd_config";
-    enter_cmd("[ `grep \"^PermitRootLogin *yes\" $sshd_config_file | wc -l` -gt 0 ] || (echo 'PermitRootLogin yes' >>$sshd_config_file; systemctl restart sshd)");
+    enter_cmd("[ `grep \"^PermitRootLogin *yes\" $sshd_config_file | wc -l` -gt 0 ] || (echo 'PermitRootLogin yes' >>$sshd_config_file; systemctl restart sshd)", wait_still_screen => 5);
```

Comment:
I guess there will be still chances that permit_root_ssh_in_sol fail since typing commands in sol console usually have errors.  It won't be fully solved until we switch to autoyast host installation.  We can prioritize task if it indeed is quite unstable here. HDYT?

Reply:
Good idea. It is said that this sshd configuration is required by SLE-next as well.

Reply:
@Julie-CAO  Glad to know that you have the same idea :). Do we have ticket tracking this? Or shall we create a new one for it?

Reply:
Are you talking about a ticket to allow root ssh login via autoyast host installation? If so, there has not been this ticket yet.

Reply:
> @Julie-CAO Glad to know that you have the same idea :). Do we have ticket tracking this? Or shall we create a new one for it?

I had a search, seems no ticket for it yet, so create https://progress.opensuse.org/issues/124149.